docs/configuration/worker.md

```diff
@@ -161,6 +161,7 @@ license: |
 | celeborn.worker.shuffle.partitionSplit.min | 1m | false | Min size for a partition to split | 0.3.0 | celeborn.shuffle.partitionSplit.min | 
 | celeborn.worker.sortPartition.indexCache.expire | 180s | false | PartitionSorter's cache item expire time. | 0.4.0 |  | 
 | celeborn.worker.sortPartition.indexCache.maxWeight | 100000 | false | PartitionSorter's cache max weight for index buffer. | 0.4.0 |  | 
+| celeborn.worker.sortPartition.memoryCheck.bypass | false | false | Bypass memory check in partition files sorter. If your worker has quite large direct memory and same Celeborn cluster supports Spark and Flink, enable this setting will be helpful. | 0.6.0 |  | 
```

Comment:
docs of `celeborn.worker.partitionSorter.directMemoryRatioThreshold` should be updated too.

Reply:
Sounds good. I'll change this PR.